bbp/default.nix

```diff
@@ -232,6 +238,18 @@ let
 
         };
 
+        pynrrd = callPackage ./nse/pynrrd {};
+        equation = callPackage ./nse/equation {};
+        numpy_1_13_1 = callPackage ./nse/numpy/1.13.1.nix {};
```

Comment:
Yep, I plan to make pull requests to the upstream nixpkgs for all the generic dependencies but until they are merged there and merged to bbp repo, I plan to have them under nse packages.